src/middleware/client.js

```diff
@@ -72,7 +73,7 @@ module.exports = function hullClientMiddlewareFactory(Client, { hostSecret, fetc
       const { message, config } = req.hull;
       const { organization, ship: id, secret } = config;
       if (organization && id && secret) {
-        const client = req.hull.client = new Client({ id, secret, organization });
+        const client = req.hull.client = new Client(_.merge({ id, secret, organization }, additionalConfig));
```

Comment:
What about naming this `clientConfig` or `hullConfig` instead ?
With `additionalConfig` it's not clear that it's configuration for the hull client...